### ISSUE TITLE
RPM build takes .o files from bzip2

### ISSUE THREAD

**Issue by thorgrin** [VUTBR/libnf#9]
I've run into following issue while doing rpm build on Fedora 24. I've done
```
./configure
make
make rpm
```
and the build failed on 
```
libtool: link: gcc -ggdb -I../include -I../nfdump/bin -I../src/ffilter -I../bzip2 -fPIC -DLNF_THREADS -DNSEL -Wl,-z -Wl,relro -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -o libnf-info libnf-info.o  ../src/.libs/libnf.a -lresolv -lrt -lpthread
/usr/bin/ld: ../src/.libs/libnf.a(bzlib.o): relocation R_X86_64_32S against `BZ2_crc32Table' can not be used when making a shared object; recompile with -fPIC
../src/.libs/libnf.a: error adding symbols: Bad value
collect2: error: ld returned 1 exit status
Makefile:584: recipe for target 'libnf-info' failed
```

However, by doing
```
make clean
make rpm
```

I was  able to complete successfully. The problem is that the fedora rpm builds apply some hardening (ASLR support is my best guess) which requires all code to be position independent. However, the `make rpm` copies some .o files from bzip2 dir created by previous `make`. These files are build with different flags and cause the above described issue.

**Comment by cejkato2** [comment]
I think this might be a duplicate of #3.

**Comment by thorgrin** [comment]
The #3 seems like another problem which solves the build issue in another way. However, the solution just masks this problem, which should be resolved as well.

**Comment by tpoder1** [comment]
To quick workaround for this issue is additional run of make clean: 

 wget http://libnf.net/packages/libnf-1.25.tar.gz
 tar xzf libnf-1.25.tar.gz
 cd libnf-1.25
 make
 make clean 
 make rpm

Issue is fixed in commit b6cd96c.
